### DISCUSSION TITLE
Hope to release the Android APK version for users to download by themselves希望可以发布安卓APK版本供用户自行下载

### DISCUSSION THREAD

**Discussion by solider245** [omnivore-app/omnivore#1849] [Ideas]
由于无法安装谷歌市场，所以无法安装安卓版本。

如果可以上传APK版本，那么我们就可以自行下载了。

Because the Google market cannot be installed, the Android version cannot be installed.

If you can upload the APK version, then we can download it by ourselves.

**Comment by satindar** [comment]
@solider245 Hi we were planning to add the app to F-Droid: https://f-droid.org

Would you be able to install the app from there?

**Reply by solider245** [reply]
谢谢，可以，方便的话最好直接上传到Github上，更加方便！
Thank you, yes, it is better to upload it directly to github, it is more convenient!

**Reply by mangkoran** [reply]
Any update regarding this? I (and many others) also would like to be able to install APK directly. F-Droid is good, but it have some requirements that need to be fulfilled. In contrast, publishing the APK in this repo shouldn't be that hard (unless there are some sort of things that I missed).
